### DISCUSSION TITLE
手游「绝区零」体验评价 - CafeBlogs

### DISCUSSION THREAD

**Discussion by giscus** [CafeKiller/CafeKiller.github.io#3] [General]
# 手游绝区零短期体验(持续更新中) - CafeBlogs

最近玩了一下米哈游最近手游《绝区零》感觉有挺多想唠叨唠叨的，所以这是一篇体验吐槽？锐评？碎碎念？总之纯玩家、纯主观、纯体验

https://blog.cafekiller.fun/games/games240707/

**Comment by EternityWills** [comment]
666

**Comment by CafeKiller** [comment]
**1.4版本，绝区零重新开服啦，正在体验中**

**Comment by EternityWills** [comment]


**Reply by CafeKiller** [reply]
需自行移至仓库 `discussions` 模块，找到相关话题，点击 `Unsubscribe`，即可取消话题邮件通知

> [仓库 discussions 链接](https://github.com/CafeKiller/CafeKiller.github.io/discussions/3)
